demo-private.html

```diff
@@ -44,12 +45,19 @@
 </head>
 <body unresolved>
 
+<google-signin
+  clientId="1054047045356-j8pgqgls9vdef3rl09hapoicumbte0bo.apps.googleusercontent.com"
```

Comment:
The Spreadsheets feeds API is old-school gdata. It doesnt require a clientid and isn't a js client api.